resources/options.html

```diff
@@ -43,120 +43,79 @@
-                        <label for="hashUsername"> Encrypt your username. </label>
-                    </li>
-                    <li>
-                        <input type="checkbox" id="hashRepo" checked>
-                        <label for="hashRepo"> Encrypt the names of repositories you're working in. </label>
-                    </li>
-                    <li>
-                        <input type="checkbox" id="hashFile">
-                        <label for="hashFile"> Encrypt the names of the files you're working on. </label>
-                     </li>
-                </ul>
-            </div>
-        </div>
-    </div>
 
-    <div class="col s12 m6">
-        <div class="card" id="doNotWatch">
-            <div class="card-content">
-                <span class="card-title">Tracking selection</span> <br>
-                <p>
-                    The checked elements and events will be tracked while Octopeer is running.
-                </p>
-                <ul id="doNotWatchSubOptions">
-                    <li>
-                        <input type="checkbox" id="dnwScreen" checked>
-                        <label for="dnwScreen">Elements on the screen. </label>
-                    </li>
-                    <li>
-                        <input type="checkbox" id="dnwHover" checked>
-                        <label for="dnwHover">Elements I hover over with the mouse. </label>
-                    </li>
-                    <li>
-                        <input type="checkbox" id="dnwComments" checked>
-                        <label for="dnwComments">Comments on my Pull Request. </label>
-                    </li>
-                    <li>
-                        <input type="checkbox" id="dnwKeyboardShortcut" checked>
-                        <label for="dnwKeyboardShortcut">The keyboard shortcuts I use. </label>
-                    </li>
-                </ul>
+        <div class="col s12 m6" id="mouse">
+            <div class="card">
+                <div class="card-content">
+                    <span class="card-title">Mouse</span> <br>
+                    <p>
+                        While I'm reviewing pull requests on GitHub, the following actions involving my mouse may be tracked:
+                    </p>
+                    <ul id="mouseSubOptions">
+                        <li>
+                            <input type="checkbox" id="mouseHover" checked>
+                            <label for="mouseHover"> what elements I hover over.</label>
+                        </li>
+                        <li>
+                            <input type="checkbox" id="mouseClick" checked>
+                            <label for="mouseClick"> what elements I click on. </label>
+                        </li>
+                        <li>
+                            <input type="checkbox" id="mouseScrolling" checked>
+                            <label for="mouseScrolling"> how I use the scroll bar. </label>
```

Comment:
People can also use the scroll wheel? :P Maybe make this a bit more general, like "when I scroll through the page"?